### PR TITLE
Add Stellastra to the monitors list

### PR DESCRIPTION
**Pull request by stellastrareviews**
Certificate Transparency Log Addition

### CODE REVIEW

File: data/monitors.json [hunk 1]
```diff
@@ -54,6 +54,13 @@
       "description": "Monitor domains for expiring and unauthorized SSL certificates and get notified if there’s a problem.",
       "logo": "sslmate.svg"
     },
+    
+    {
+      "name": "Stellastra",
+      "url": "https://stellastra.com/solution/certificate-transparency-log-monitor/",
+      "description": "Monitor for Unauthorized, Expiring, and Maliciously Issued SSL/TLS Certificates.",
+      "logo": "stellastra.svg"
```
Comment (roger2hk) [inline data/monitors.json:62]: Please add the svg file in the same pull request.

**Review by roger2hk — CHANGES_REQUESTED** [review]

**Review by roger2hk — CHANGES_REQUESTED** [review]
Please follow the steps to sign the CLA.

https://github.com/google/certificate-transparency-community-site/pull/70/checks?check_run_id=24943494524

**Review by stellastrareviews — COMMENTED** [review]
CLA Done

**Review by roger2hk — APPROVED** [review]

**Comment by roger2hk** [comment]
The change is now live in https://certificate.transparency.dev/monitors/.

**Pull request merged**